### ISSUE TITLE
Function perceptual_forward in wav2lip.py

### ISSUE THREAD

**Issue by Augnine** [Rudrabha/Wav2Lip#76]
i am confused with the code below in function perceptual_forward in wav2lip.py

`false_pred_loss = F.binary_cross_entropy(self.binary_pred(false_feats).view(len(false_feats), -1),  torch.ones((len(false_feats), 1)).cuda())`

I think the second variable should be torch.zeros instead of torch.ones in `F.binary_cross_entropy`

**Comment by prajwalkr** [comment]
No, the `perceptual_forward` function is used when training the generator. When training the generator, you need to compute the loss which says how far the generated images are from "real", i.e., you need to compute the loss with the real label, which is `torch.ones`.
